google/cloud/vision/entity.py

```diff
@@ -55,7 +55,9 @@ def from_api_repr(cls, response):
         :rtype: :class:`~google.cloud.vision.entiy.EntityAnnotation`
         :returns: Instance of ``EntityAnnotation``.
         """
-        bounds = Bounds.from_api_repr(response['boundingPoly'])
+        bounds = []
+        if 'boundingPoly' in response:
+            bounds = Bounds.from_api_repr(response['boundingPoly'])
```

Comment:
so I actually just switched [this](https://github.com/GoogleCloudPlatform/google-cloud-python/pull/2237/commits/e4d49744450945b28b4711e9469bb4cd0bacedbf) to my suggestion, it seems a lot cleaner this way.